### ISSUE TITLE
relation "attachments" already exists

### ISSUE THREAD

**Issue by khanakia** [codedor/filament-media-library#18]
When I try to run the `php artisan migrate` it gives below error. I have verified there were no attachment tables in my database before.
```sh

   INFO  Running migrations.  

  2022_08_03_120355_create_attachments_table ................................................................................. 8ms DONE
  2022_08_03_120356_create_attachment_tags_table ............................................................................. 5ms DONE
  2022_08_03_120357_create_attachment_attachment_tags_table .................................................................. 5ms DONE
  2023_04_27_120359_create_attachment_formats ................................................................................ 4ms DONE
  2023_12_09_042526_2022_08_03_120355_create_attachments_table ............................................................... 1ms FAIL

   Illuminate\Database\QueryException 

  SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "attachments" already exists (Connection: pgsql, SQL: create table "attachments" ("id" uuid not null, "name" varchar(255) not null, "extension" varchar(255) not null, "mime_type" varchar(255) not null, "md5" varchar(255) not null, "type" varchar(255) not null, "size" bigint not null, "width" bigint null, "height" bigint null, "disk" varchar(255) not null, "translated_name" json null, "alt" json null, "caption" json null, "created_at" timestamp(0) without time zone null, "updated_at" timestamp(0) without time zone null))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:801
    797▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    798▕                 );
    799▕             }
    800▕ 
  ➜ 801▕             throw new QueryException(
    802▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    803▕             );
    804▕         }
    805▕     }

      +9 vendor frames 

  10  database/migrations/2023_12_09_042526_2022_08_03_120355_create_attachments_table.php:16
      Illuminate\Support\Facades\Facade::__callStatic("create")
      +25 vendor frames 

  36  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

**Comment by khanakia** [comment]
I think the error is because of this command it duplicates all database tables again so laravel first migrates from the vendor library itself and then tries to migrate again from the duplicated files in `database/migrations` the same tables

So I believe you should modify the package and stop publishing the migration files as laravel handles it automatically now
```sh
git:(main) ✗ php artisan vendor:publish --provider "Codedor\MediaLibrary\Providers\MediaLibraryServiceProvider"

   INFO  Publishing assets.  

  Copying file [vendor/codedor/filament-media-library/config/filament-media-library.php] to [config/filament-media-library.php]  DONE
  Copying directory [vendor/codedor/filament-media-library/resources/views] to [resources/views/vendor/filament-media-library]  DONE
  Copying file [vendor/codedor/filament-media-library/database/migrations/2022_08_03_120355_create_attachments_table.php] to [database/migrations/2023_12_09_042526_2022_08_03_120355_create_attachments_table.php]  DONE
  Copying file [vendor/codedor/filament-media-library/database/migrations/2022_08_03_120356_create_attachment_tags_table.php] to [database/migrations/2023_12_09_042527_2022_08_03_120356_create_attachment_tags_table.php]  DONE
  Copying file [vendor/codedor/filament-media-library/database/migrations/2022_08_03_120357_create_attachment_attachment_tags_table.php] to [database/migrations/2023_12_09_042528_2022_08_03_120357_create_attachment_attachment_tags_table.php]  DONE
  Copying file [vendor/codedor/filament-media-library/database/migrations/2023_04_27_120359_create_attachment_formats.php] to [database/migrations/2023_12_09_042529_2023_04_27_120359_create_attachment_formats.php]  DONE
  Copying directory [vendor/codedor/filament-media-library/resources/lang] to [lang/vendor/filament-media-library]  DONE


```

**Comment by jyrkidn** [comment]
Fixed this in v1.0.9, we won't publish the migrations anymore, but run them straight from the package
